cartographer_rviz/cartographer_rviz/drawable_submap.cc

```diff
@@ -162,19 +167,36 @@ void DrawableSubmap::SetAlpha(const double current_tracking_z) {
       target_alpha == 0.f || target_alpha == 1.f) {
     current_alpha_ = target_alpha;
   }
-  ogre_slice_.SetAlpha(current_alpha_);
+  for (auto& slice : ogre_slices_) {
+    slice->SetAlpha(current_alpha_);
+  }
   display_context_->queueRender();
 }
 
+void DrawableSubmap::SetSliceVisibility(size_t slice_index, bool visible) {
```

Comment:
instead of if use CHECK? it seems a programming error if 'slice_index' is out of range.